src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIFight.java

```diff
@@ -84,9 +84,9 @@ public AbstractEntityAIFight(@NotNull final J job)
     {
         super(job);
         super.registerTargets(
-          new AITarget(IDLE, START_WORKING, 1),
+          new AITarget(IDLE, PREPARING, 1),
```

Comment:
Is start working unused now? if so remove the AITarget/function

Reply:
START_WORKING is still used for regeneration and when a weapon breaks on the guard etc. I just made it not be the initial state, because that means that after server restart the guards will always go home to their hut (`startWorkingFromOwnBuilding` is part of that state), which would potentially kill you when a server restart happens while you're rally/fighting.

In any case, Ray wanted to improve the guard AI anyway - it's a little bit "weird" at the moment, definitely not like more modern AIs are programmed. So he will probably change this too, if he wants to.

Reply:
ah its still used fine then